language-design.md

```diff
@@ -106,7 +254,19 @@ you do so? If not, why not?
 
 **Response**
 
+We believe that these two experiences of natural languages are not at odds with one another. Pavlus' statement that natural language replacements are more readable is correct. A simple example is the readability of for loops in Python and C++. When introducing for-loops to new programmers, the Python language is much more intuitive in expressing the actions of the program. We do recognize that _natural_ language as a programming language is extraordinarily difficult since natural language relies both on the context of speaker and the context of the listener. Trying to fit a natural language into the confines of a strict interpretation is the same issue that automated translators have. (As a fun aside, try to translate some phrases in Google Translate and then translate them back into English). Even the interpretation of simple phrases is not one-to-one and is often many-to-many, unideal for programming languages and their interpreters. We posit that natural language in a programming languages _is_ beneficial, but to an extent. We are unsure where the boundary lies between too much and too little but we will discuss this later. Historically, we find that programming languages generally neglect the clarity that natural language can provide.
```

Comment:
We talked about this a lot too, that natural languages are very expressive, and that writing a parser that could handle all of the different ways a natural language works would be... interesting.  Furthermore, if people had access to all conceivable ways of expressing something in the English language as ways to express a programming theme, no code written in that language would be consistent.  Alternatively, in an attempt to combat that issue, users would limit the language by convention, and we wouldn't have natural language anymore, anyway.